ci/release_notes.md

```diff
@@ -1,4 +1,3 @@
-##  Updated packages
+##  New options
 
-
-Updates docker distribution binary from 2.5.1 to 2.6.2.
\ No newline at end of file
+`docker.cache.disabled: true` disables the blobdescriptor cache.  This can be useful in CI/CD environments using this boshrelease as a pullthrough cache due to a race condition that can occur if an image is pulled before it finishes uploading to the upstream registry.
```

Comment:
Don't trash your release notes for the upgrade to 2.6.2! 😱

Reply:
Ah ok thanks! I wasnt sure if CI keys the release notes off of PR merges.  I'll update the release notes to include both.  

Also for reference, this was tested with the 2.6.2 binaries.

Reply:
Perfect.  You probably just need to merge/rebase master into your feature branch.